kerl

```diff
@@ -655,6 +656,32 @@ maybe_patch_darwin() {
     fi
 }
 
+maybe_patch_osx_catalina() {
+    if [[ $(uname -r) == "19"* ]]; then
+    cd "$ERL_TOP"
+        
+    sed -i '.bak' ':t
```

Comment:
All of the other `apply_` functons use `patch -p0 < END_PATCH` and embed a diff right in kerl. I would like to keep doing that for this patch too, please.